src/lib/info/TrackInfoPopup.svelte

```diff
@@ -313,6 +370,7 @@
         previousScope = hotkeys.getScope();
         hotkeys.setScope("track-info");
         document.addEventListener("paste", onPaste);
+        document.getElementsByClassName("artwork-container").focus();
```

Comment:
getElementsByClassName returns an array, so you need to call `focus` on the first element

Reply:
I'm removing the line, I think it's a debug line, a bad one at that...